easybuild/tools/modules.py

```diff
@@ -1110,6 +1128,7 @@ def update(self):
 
 class EnvironmentModulesTcl(EnvironmentModulesC):
     """Interface to (Tcl) environment modules (modulecmd.tcl)."""
+    NAME = "ancient Tcl-only Environment Modules"
```

Comment:
nothing shorter came to mind?

Reply:
I'm open to suggestions. `Environment Modules` is not specific enough, and this type of modules tool doesn't have proper versioning.

We should probably deprecate support for this one too, since I doubt anyone is still using it...
I'll do that in a separate PR once this one is merged.